tests/test_routes_project_tasks_aggregate.py

```
@@ -0,0 +1,374 @@
+    ids = _project_ids(resp)
+    assert pid_a in ids
+    assert pid_b not in ids
+    # A's board carries A's task; B's task is nowhere in the aggregate.
+    assert tid_a in _task_ids(resp, pid_a)
+    assert tid_b not in {t["id"] for it in resp.json()["items"] for t in it["tasks"]}
+
+
+@pytest.mark.asyncio
+async def test_owner_aggregate_empty_when_no_projects(members):
+    """A caller with no projects gets an empty aggregate, not an error."""
+    alice, bob = members
```

Ruff (0.16.3)

[warning] Unpacked variable `bob` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)